crates/jit/src/instantiate.rs

```diff
@@ -19,6 +19,10 @@ use wasmtime_environ::{
     CompileError, DataInitializer, DataInitializerLocation, Module, ModuleEnvironment,
     ModuleSyncString,
 };
+#[cfg(feature = "profiling")]
+use wasmtime_profiling::JitDumpAgent;
+#[cfg(not(feature = "profiling"))]
+use wasmtime_runtime::JitDumpAgent;
```

Comment:
it might be good idea to just import `JitDumpAgent` at wasmtime_runtime and don't use `#[cfg()]`. (See also proposal about ProfilerInfoAgent)